packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -14,18 +24,20 @@
 }
 
 :host([scale="s"]) {
-  --calcite-toggle-spacing: theme("spacing.2");
-  --calcite-internal-input-text-input-padding-inline-end: #{get-trailing-text-input-padding(theme("spacing.4"))};
+  --calcite-toggle-spacing: var(--calcite-spacing-sm);
+  --calcite-internal-input-text-input-padding-inline-end: #{get-trailing-text-input-padding(var(--calcite-spacing-lg))};
 }
 
 :host([scale="m"]) {
-  --calcite-toggle-spacing: theme("spacing.3");
-  --calcite-internal-input-text-input-padding-inline-end: #{get-trailing-text-input-padding(theme("spacing.6"))};
+  --calcite-toggle-spacing: var(--calcite-spacing-md);
+  --calcite-internal-input-text-input-padding-inline-end: #{get-trailing-text-input-padding(var(--calcite-spacing-xxl))};
 }
 
 :host([scale="l"]) {
-  --calcite-toggle-spacing: theme("spacing.4");
-  --calcite-internal-input-text-input-padding-inline-end: #{get-trailing-text-input-padding(theme("spacing.8"))};
+  --calcite-toggle-spacing: var(--calcite-spacing-lg);
```

Comment:
should this ` --calcite-toggle-spacing` be renamed to be internal? Like ` --calcite-internal-date-picker-toggle-spacing`?